modules/3d/src/rgbd/odometry_frame_impl.cpp

```diff
@@ -0,0 +1,250 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/core/ocl.hpp>
+#include <opencv2/core/mat.hpp>
+#include <opencv2/imgproc/types_c.h>
```

Comment:
> types_c.h

Avoid using of legacy C API in new code.

Reply:
done